### ISSUE TITLE
[HL2] Lost Coast is missing its menu title and loading screen.

### ISSUE THREAD

**Issue by Malt33** [ValveSoftware/Source-1-Games#6712]
When selecting "Lost Coast" from the extras tab, the main menu will display "Half-Life 2" instead of the proper title.
The loading screen for the Lost Coast chapter (the blurred image of the background map) is also incorrect.

Since the Anniversary Update, the contents of the **"hl2_complete"** folder override all other files, _even custom content_.

The Half-Life 2 background image is inside that folder and completly replaces the Lost Coast one.
That's because Lost Coast's background files have the same name as those from Half-Life 2 (background01).

The **"hl2_complete"** folder doesn't seem to allow custom content, meaning that these files (like chapter icons) can't be edited for any of the games.

**Comment by dwartbg** [comment]
Yeah, this is what I noticed too.  It will be great if they update it with the proper title.

**Comment by SirFritz** [comment]
It also doesn't update your friends activity status (don't know the correct name).  It just lists you as being "In main menu" instead of in lost coast specifically.

**Comment by RiverHornet3209** [comment]
If lost coast is launched directly from the steam library, the main menu displays the proper title and the proper loading background:

![Image](https://github.com/user-attachments/assets/c6a14430-f12e-4cb9-8d67-fb595ca24b7f)

![Image](https://github.com/user-attachments/assets/28bee79f-6910-4bf3-8e3e-120b3ff35def)

If lost coast is launched from the "EXTRAS" menu in Half-Life 2, the main menu displays "Half-Life 2" instead of "Half-Life 2 Lost Coast" and the loading background is incorrect:
![Image](https://github.com/user-attachments/assets/7909092b-abe1-46f1-b052-5aa8a378b4a8)

![Image](https://github.com/user-attachments/assets/1f2906a2-aa0c-42f3-90ae-6347e3170786)
